src/modules/Builder/features/ActivitySettings/ScoresAndReports/ScoresAndReports.utils.ts

```diff
@@ -17,9 +19,16 @@ export const getScoreDefaults = () => ({
 
 export const getSectionDefaults = () => ({
   name: '',
+  type: ScoreReportType.Section,
   id: uuidv4(),
   showMessage: true,
   printItems: false,
   message: '',
   itemsPrint: [],
 });
+
+export const getReportIndex = (reports: ScoreOrSection[], report: ScoreOrSection) => {
+  const entities = reports?.filter(({ type }) => type === report.type);
```

Comment:
mb use reduce instead filter  and findIndex

Reply:
done